src/client/hypertable/CephBroker.cc

```diff
@@ -200,22 +200,22 @@ void CephBroker::read(ResponseCallbackRead *cb, uint32_t fd, uint32_t amount) {
 
   if (!m_open_file_map.get(fd, fdata)) {
     char errbuf[32];
-    sprintf(errbuf, "%d", fd);
+    sprintf(errbuf, "%u", fd);
```

Comment:
we'd better to print an uint32_t using the macro of `PRIu32` instead of `%u`, which works on platforms with 32bit integers but not on some other platforms.